golang/internal/lab4/lab4.go

```diff
@@ -0,0 +1,26 @@
+package lab4
+
+import (
+	"math"
+)
+
+func Calc(x float64) float64 {
+	return math.Pow((math.Pow(math.Asin(x), 2) + math.Pow(math.Acos(x), 4)), 3)
+}
+
+func Primer1(xn, xk, deltaX float64) []float64 {
+	results := make([]float64, 0)
```

Comment:
укажите cap чтобы не пересоздавать 
```suggestion
	results := make([]float64, 0, (xk-xn)/deltaX + 1)
```